contrib/cirrus/ts.awk

```diff
@@ -0,0 +1,14 @@
+BEGIN {
```

Comment:
Would you consider adding a header comment, so someone poking in the source code can know what this file does and who invokes it and why?

Reply:
yep, good idea.